scripts/e2e/ios-devices.js

```diff
@@ -0,0 +1,41 @@
+const DEFAULT_APPLE_DEVICE = 'iPhone 17';
+const DEFEAULT_IOS_VERSION = 'iOS 26.2';
+
+/**
+ * @return {string}
+ */
+function getAppleDevice() {
+  return process.env.RNS_E2E_APPLE_SIM_NAME || DEFAULT_APPLE_DEVICE;
```

Comment:
We may consider adding some extra validation and define the types of devices that we're supporting for e2e automation, e.g. for now I'm seeing a use-case for passing `iPhone` or `iPad`, but when someone passes `watchOS`, we should have some error handling prepared